installer/templates/phx_single/mix.exs

```diff
@@ -54,7 +54,7 @@ defmodule <%= @app_module %>.MixProject do
       {:telemetry_poller, "~> 1.0"},<%= if @gettext do %>
       {:gettext, "~> 0.20"},<% end %>
       {:jason, "~> 1.2"},
-      {:plug_cowboy, "~> 2.5"}
+      {<%= inspect @web_adapter_app %>, <%= inspect @web_adapter_version %>}
```

Comment:
I wonder (as in I'm legitimately not sure) if this would be better expressed via a fixed `">= 0.0.0"` here, and letting phoenix itself declare specific version dependencies via an `:optional` dependency on bandit (it already has one on plug_cowboy). This would keep the onus on Phoenix to manage server dep versions, as we [just did](https://github.com/phoenixframework/phoenix/pull/5030/files#diff-dfa6f4ed74c90e5d4fda283d547d366586e690387289bcfd473e3fa5f9ace308R83) for WebSock support.